src/components/InfoViewer/InfoViewer.tsx

```diff
@@ -15,6 +15,7 @@ interface InfoViewerProps {
     size?: 's';
     className?: string;
     multilineLabels?: boolean;
+    nullOnEmptyInfo?: boolean;
```

Comment:
It seems it will be more useful if you add `renderEmptyState` prop - a function that returns a component. If prop is not passed, you return null.

Reply:
By default component returns title and no data error message. But the idea with the functionÏ looks very good, this prop may be very useful in future